setup.py

```diff
@@ -23,58 +28,61 @@
 License :: Other/Proprietary License
```

Comment:
Is this correct?

Reply:
Yes, as I understand it, it means we have a custom license, not BSD/MIT/GPL/etc.